RELEASE_NOTES.md

```diff
@@ -2,13 +2,16 @@
 
 ## Summary
 
-This release adds support for pagination in the dispatch list request.
+This release includes a new feature for pagination support in the dispatch list request as well as usage of the base-client for setting up the channel and client configuration.
 
 ## Upgrading
 
 - The `Client.list()` function now yields a `list[Dispatch]` representing one page of dispatches
+- `Client.__init__` no longer accepts a `grpc_channel` argument, instead a `svc_addr` argument is required.
```

Comment:
```suggestion
- `Client.__init__` no longer accepts a `grpc_channel` argument, instead a `server_url` argument is required.
```
![image](https://github.com/user-attachments/assets/f0f6b411-435d-4786-8ef5-7facf4228802)

Reply:
that's what you meant :laughing: 
I updated the release notes while rebasing ;)-